### PR TITLE
tests(ticdc): add multi topic kafka test

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/4423

### What is changed and how it works?

- add multi topic kafka test(canal-json)
- mount the tests dir into docker
- refine the stupid code

### Check List 

Tests 

 - Integration test

Code changes

None

Side effects

None

Related changes

None

### Release note 

```release-note

None
```

### CODE REVIEW

**Review by zhaoxinyu — APPROVED** [review]

File: cdc/owner/changefeed.go [hunk 1]
```diff
@@ -197,6 +197,7 @@ func (c *changefeed) tick(ctx cdcContext.Context, state *orchestrator.Changefeed
 	// and we need to use the latest table names.
 	if c.currentTableNames == nil {
 		c.currentTableNames = c.schema.AllTableNames()
+		log.Debug("changefeed current table names updated", zap.Any("tables", c.currentTableNames))
```
Comment (overvenus) [inline cdc/owner/changefeed.go:200]: Could you add `changefeed` field?

**Review by overvenus — APPROVED** [review]
Rest LGTM

File: cdc/owner/changefeed.go (hunk 1 above)
Comment (hi-rustin) [reply cdc/owner/changefeed.go]: Good catch! 
Thanks for your review!  💚 💙 💜 💛 ❤️

**Pull request merged**